core/container/src/main/java-templates/org/wildfly/swarm/SwarmInfo.java

```diff
@@ -0,0 +1,15 @@
+package org.wildfly.swarm;
+
+/**
+ * @author Ken Finnigan
+ */
+public interface SwarmInfo {
+
+    final String VERSION = "${project.version}";
+
+    final String GROUP_ID = "${project.groupId}";
+
+    default boolean isProduct() {
```

Comment:
Otherwise you won't be able to call it without implementing this interface somehow